<launch>
  <node name="hexapod_teleop_key" pkg="hexapod_teleop" type="hexapod_teleop_key" output="screen">
    <param name="pos_scale" value="1.0" type="double"/>
    <param name="rot_scale" value="1.0" type="double"/>
  </node>
</launch>